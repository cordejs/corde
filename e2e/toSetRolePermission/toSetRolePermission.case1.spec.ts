import Utils from "../testUtils";
import { runCLI } from "../cliRunner";

it("should set role permissions", async () => {
  const command = Utils.buildCommandWithConfigPath("toSetRolePermission", "bot_case1.spec.ts");
  const [mockProcess, stdout] = await runCLI(command);
  expect(stdout).toMatchSnapshot();
  expect(mockProcess).toBeCalledWith(0);
});
