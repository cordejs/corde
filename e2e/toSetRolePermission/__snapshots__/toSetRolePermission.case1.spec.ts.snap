// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should set role permissions 1`] = `
"FileError: No test file was found in the path './e2e/toSetRolePermission/__cordeTest__/bot_case1.spec.ts'
"
`;
